### PR TITLE
filter overflow fix

### PR DESCRIPTION
**Pull request by sherzinger**
closes ada-discovery/ada-issues#144

### CODE REVIEW

File: app/views/filter/conditionPanel.scala.html [hunk 1]
```diff
@@ -11,9 +11,9 @@
     @filter.map(_.conditions).getOrElse(Nil).zipWithIndex.map { case (condition, index) =>
     <div id="condition-full@index" class="filter-part condition-full">
         <span class="condition">
-            <span class="label label-primary" id="fieldLabel">@condition.fieldLabelOrElseName</span>
-            <span id="conditionType"><b>@conditionTypeToReadableOperator(condition.conditionType)</b></span>
-            <span class="label label-primary" id="value">@valueOrUndefined(condition, trim = true)</span>
+            <span class="filter-label label label-primary" id="fieldLabel">@condition.fieldLabelOrElseName</span>
```
Comment (peterbanda) [inline app/views/filter/conditionPanel.scala.html]: I would maybe keep the field label (not the condition type or value) bold or a bit larger to make it distinguishable... but first I will check how it looks like

Comment (peterbanda) [reply app/views/filter/conditionPanel.scala.html]: This looks fine as it is

File: public/javascripts/multiFilter.js [hunk 2]
```diff
@@ -364,9 +364,9 @@ $.widget("custom.multiFilter", {
         var li = $("<li>", {id: "condition-full" + index, "class": "condition-full"});
 
         var conditionSpan = $("<span>", {"class": "condition"})
-        conditionSpan.append("<span class='label label-primary' id='fieldLabel'>" + condition['fieldLabel'] + "</span>");
-        conditionSpan.append("<span id='conditionType'><b> " + condition['conditionType'] + " </b></span>");
-        conditionSpan.append("<span class='label label-primary' id='value'>" + condition['value'] + "</span>");
+        conditionSpan.append("<span class='filter-label label label-primary' id='fieldLabel'>" + condition['fieldLabel'] + "</span>");
+        conditionSpan.append("<span id='filter-label conditionType'><b> " + condition['conditionType'] + " </b></span>");
```
Comment (peterbanda) [inline public/javascripts/multiFilter.js]: maybe `id='conditionType' class='filter-label'`

File: public/stylesheets/main.css [hunk 3]
```diff
@@ -418,6 +422,7 @@ a.carousel-control.right{
 
 #conditionPanel {
     display: flex;
+    overflow: scroll;
```
Comment (peterbanda) [inline public/stylesheets/main.css]: I've pulled the code and I see horizontal and vertical scrolls in a condition panel (by default) even they are not needed.. will send a screenshot

Comment (sherzinger) [reply public/stylesheets/main.css]: Can you please show me? I can't reproduce it here
Might be a linux thing. Scrollbars are handled by the OS

**Pull request merged**